fdbserver/masterserver.actor.cpp

```diff
@@ -778,6 +781,15 @@ ACTOR Future<vector<Standalone<CommitTransactionRef>>> recruitEverything(Referen
 	self->tLogGroupCollection->loadState(self->tLogGroupRecoveredState.get());
 	self->tLogGroupCollection->recruitEverything();
 
+	// Restore storage team to TLogGroup assignments. This will make sure that
+	// across recovery, we don't assign a different TLogGroup to a team. It
```

Comment:
I am confused. I thought TLogGroup is created per epoch and does not work across epochs.